src/CachePeer.cc

```diff
@@ -131,3 +136,421 @@ operator <<(std::ostream &os, const CachePeer &p)
     return os << p.name;
 }
 
+/**
+ * utility function to prevent getservbyname() being called with a numeric value
+ * on Windows at least it returns garage results.
+ */
+static bool
+isUnsignedNumeric(const char *str, size_t len)
```

Comment:
I copied all this code 'as-is' from cache_cf.cc, without any refactoring. Should we polish it?